code/modules/mob/dead/observer/observer.dm

```diff
@@ -206,12 +206,25 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 
 // This is the ghost's follow verb with an argument
 /mob/dead/observer/proc/ManualFollow(atom/movable/target)
-	if(target && target != src)
-		if(orbiting && orbiting == target)
-			return
+	if (!istype(target))
+		return
+
+	var/icon/I = icon(target.icon,target.icon_state,target.dir)
+
+	var/orbitsize = (I.Width()+I.Height())/2
+	orbitsize -= (orbitsize/32)*8

```

Comment:
Nope. I had ghost orbit set to 24, (32-8), I made this to make it scale better in the same way.

Other wise I would have to do (orbitsize/world.icon_size)*(world.icon_size*0.25), and that would lead to the exact same effect in all cases.